packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -131,9 +131,7 @@ const ProofOfIdentityContainer = ({
     } else if (
         !identity_last_attempt ||
         // Prioritise verified status from back office. How we know this is if there is mismatch between current statuses (Should be refactored)
-        (identity_status === identity_status_codes.verified && identity_status !== identity_last_attempt.status) ||
-        // Prioritise pending status from back office. We can't rely on last attempt as it doesn't get updated after manual upload (Should be refactored)
-        identity_status === identity_status_codes.pending
```

Comment:
Is last attempt getting updated after manual upload is done now?

Reply:
Yes. Manual info is now added to last_attempt